# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../gzip/glibc.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- gzip-1.3.12/lib/utimens.h.pom	2007-02-23 19:25:21.000000000 +0100
+++ gzip-1.3.12/lib/utimens.h	2007-06-04 12:18:29.000000000 +0200
@@ -1,3 +1,3 @@
 #include <time.h>
-int futimens (int, char const *, struct timespec const [2]);
+int futimens_oi (int, char const *, struct timespec const [2]);
 int utimens (char const *, struct timespec const [2]);
--- gzip-1.3.12/lib/utimens.c.pom	2007-01-18 09:33:34.000000000 +0100
+++ gzip-1.3.12/lib/utimens.c	2007-06-04 12:18:23.000000000 +0200
@@ -75,7 +75,7 @@ struct utimbuf
    Return 0 on success, -1 (setting errno) on failure.  */
 
 int
-futimens (int fd ATTRIBUTE_UNUSED,
+futimens_oi (int fd ATTRIBUTE_UNUSED,
 	  char const *file, struct timespec const timespec[2])
 {
   /* Some Linux-based NFS clients are buggy, and mishandle time stamps
@@ -185,5 +185,5 @@ futimens (int fd ATTRIBUTE_UNUSED,
 int
 utimens (char const *file, struct timespec const timespec[2])
 {
-  return futimens (-1, file, timespec);
+  return futimens_oi (-1, file, timespec);
 }
--- gzip-1.3.12/gzip.c.pom	2007-06-04 12:17:29.000000000 +0200
+++ gzip-1.3.12/gzip.c	2007-06-04 12:17:59.000000000 +0200
@@ -1643,7 +1643,7 @@ local void copy_stat(ifstat)
 	}
       }
 
-    if (futimens (ofd, ofname, timespec) != 0)
+    if (futimens_oi (ofd, ofname, timespec) != 0)
       {
 	int e = errno;
 	WARN ((stderr, "%s: ", program_name));

